### PR TITLE
add some more clips

### PR DESCRIPTION
**Pull request by pjeweb**
add some more clips to the ambassadors

### CODE REVIEW

**Comment by pjeweb** [comment]
preview: https://alveusgg-website-git-pjeweb-more-clips-alveus.vercel.app/ambassadors/

File: src/ambassadors/core.ts [hunk 1]
```diff
@@ -385,6 +396,12 @@ const ambassadors = {
       "He is an ambassador for the exploitation of wildlife in the pet trade and the fur trade.",
     clips: [
       { id: "670RYU2Xw98", caption: "Fenn Digs Up A Friend In The Enclosure" },
+      { id: "awX8PnOfhQA", caption: "Fenn Finds A Well Hidden Treasure" },
+      { id: "ScccoAPYPB0", caption: "Fenn The Fox ASMR" },
+      {
+        id: "9HbrUaQzhXI",
+        caption: "The Foxes Reunite And Fenn Couldn't Be More Excited About It",
```
Comment (MattIPv4) [inline src/ambassadors/core.ts]:
```suggestion
        caption: "The Foxes Are Excited To Be Reunited",
```
Maybe we can use a shorter caption here, as the original title is rather long?

File: src/ambassadors/core.ts [hunk 2]
```diff
@@ -407,6 +424,11 @@ const ambassadors = {
       "He is an ambassador for the exploitation of wildlife in the pet trade and the fur trade.",
     clips: [
       { id: "ClWtgsoWta0", caption: "Reed Gets Weighed With Target Training" },
+      { id: "4HZMQJoEkVA", caption: "Reed Is The Best Boy" },
+      {
+        id: "9HbrUaQzhXI",
+        caption: "The Foxes Reunite And Fenn Couldn't Be More Excited About It",
```
Comment (MattIPv4) [inline src/ambassadors/core.ts]: Ditto

**Review by MattIPv4 — DISMISSED** [review]
Otherwise, lgtm

**Review by MattIPv4 — APPROVED** [review]

**Pull request merged**